config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -72,6 +72,21 @@ config NVS_LOOKUP_CACHE_SIZE
 config ZMS_LOOKUP_CACHE_SIZE
     default 512 if ZMS
 
+# Enable Dynamic interrupts to have a possibility to reconfigure it for devices that use MPSL
+# on the application core and needs synchronization between MPSL and FLASH driver.
+if MPSL
+
+config DYNAMIC_INTERRUPTS
+    default y
+
+config DYNAMIC_DIRECT_INTERRUPTS
+    default y
+
+config MPSL_DYNAMIC_INTERRUPTS
```

Comment:
We still need it even though we do not re-enable mpsl, right?

Reply:
That's right, we need it anyway if we want to disable mpsl.